### ISSUE TITLE
Tap during clearTextFromElement failed when testing UIWebView

### ISSUE THREAD

**Issue by NSFish** [kif-framework/KIF#833]
This  can be simply reproduced by the testHost project with one extra line of code

``` ObjC
- (void)testEnteringText {
    [tester tapViewWithAccessibilityLabel:@"Input Label"];
    [tester enterTextIntoCurrentFirstResponder:@"Keyboard text"];
    [tester clearTextFromFirstResponder];//try clear the input field
}
```

Follow along the call stack, it seems like the test failed before even reach the clear part

``` ObjC
- (void)clearTextFromElement:(UIAccessibilityElement *)element inView:(UIView *)view
{
    [self tapAccessibilityElement:element inView:view];//failed

    //clear 
}
```

As I digged further,  I found that the whole "tap" process could be summarized into 4 steps:
1. When it comes to UIWebView, we are actually dealing with UIWebBrowserView, not matter where we tap on the UIWebView
2. Tap at different position on UIWebView results in same UIWebBrowserView with different **accessibilityFrame**, which will then be used to calculate the exact tap point at UIWebBrowserView
3. Using [UIView tappablePointInRect] to find if we could find a tappble UIView
4.  Tap it if we could

The real problem happened in step 2. When tapping on the "Input Label",  the accessibilityFrame was
![Correct accessibilityFrame](http://cl.ly/3E1e0D0o3R34/Correct%20accessibilityFrame.png)
which was correct. But the tap triggered by clearTextFromElement capture the wrong one
![Incorrect accessibilityFrame](http://cl.ly/2L0u0i0v2H3v/Incorrent%20accessibilibtyFrame.png)
So the test failed because it **tapped at the wrong area** on the UIWebView.

Personally I know very little about web programming, most of my experiences with UIWebView was using stringByEvaluatingJavaScriptFromString or JSContext to transfer data between js and Objective-C. This problem is a blocker to me. 
So if someone could  shed some light I would be appreciated :)

**Comment by NinjaSudo** [comment]
I've had a similar issue when testing a custom UITextField. The text gets selected, but then it fails with a timeout.

**Comment by mikelupo** [comment]
Search the forum. If memory serves I posted something on using JavaScript injection to interact with text fields in a UIWebView. 

In the case that's not your issue. Your problem may stem from a waitForViewWith check that's embedded in enterText KIF api call.

You may be able to disable that check as a workaround. 

Mike
